### DISCUSSION TITLE
CI step label-lang 변경 이후 기존 PR 적용 문제

### DISCUSSION THREAD

**Discussion by Sunjae95** [DaleStudy/leetcode-study#361] [도움요청] [answered]
[상황]
https://github.com/DaleStudy/leetcode-study/pull/354 작업 이전 draft로 올린 PR이 CI label-lang 단계에서 403 error catch가 되고 있습니다.

[방법]
https://github.com/DaleStudy/leetcode-study/pull/354 작업 이전에 개인 fork 레포지토리 브랜치 기준으로 올린 PR 작업
1. 작업이 완료된 개인 fork 레포지토리를 작업이전으로 변경
2. 개인 fork 레포지토리와 leetcode-study 레포지토리 동기화
3. fork 레포지토리에 1번에서 제거한 작업상황 반영

[문제점]
위의 3단계를 걸쳐서 했는데 깃헙에 문제가 있는지 https://github.com/DaleStudy/leetcode-study/pull/354 이전 작업의 CI가 반영되더군요.
새롭게 PR만들면될거 같은데 리뷰순서 꼬일까바 이러지도 저러지도 못하고있는데 어떻게 하면 좋을지 도움을 구하고자 합니다 :)

**Comment by bky373** [comment, marked as answer]
@Sunjae95 안녕하세요 선재님, CI 실패로 문제를 겪고 계셨군요!
음 결론을 먼저 말씀드리면, [제 작업](https://github.com/DaleStudy/leetcode-study/pull/354) 이전에 올린 선재님의 PR에서 CI 가 실패하는 건 무시하셔도 됩니다! 
해당 CI 는 PR 에 language label 를 붙여주는 건데, 실패하면 label 이 자동으로 붙지 않을 뿐 그 이상으로 선재님의 작업에, 또는 전체 프로젝트에 어떤 영향을 주거나 하지는 않습니다.
번거로우시겠지만 이 경우는 수동으로 label 추가해주시면 감사하겠습니다! 불편을 드려 죄송합니다 🙇

**Reply by bky373** [reply]
만약 Draft PR 이라면 지금 Ready for Review 로 PR 상태를 변경하셔도 되고, 새로운 PR 이라면 편하게 올리시면 될 것 같습니다!

**Reply by Sunjae95** [reply]
@bky373 안녕하세요 보람님
merge 버튼이 활성화가 되지않아서 올린글이었는데 보람님 말씀대로 language label 단계는 required가 아니군요! 
CI 구축, 답변감사합니다. close discussion 누를게요!

**Comment by DaleSeo** [comment]
@GUMUNYEONG @heozeop @JEONGHWANMIN @gitsunmin @whewchews @hyejjun @seona926 @Sunjae95 동일한 문제를 겪고 계신 것 같아서 멘션드립니다.

tl;dr: `label-lang` 체크가 실패하셔도 PR 병합 가능합니다. 해당 이슈는 이미 main 브랜치에서 조치되었기 때문에, 다음 PR에서는 동일한 문제가 발생하지 않을 것입니다.
